apps/server/src/modules/account/services/account-db.service.ts

```diff
@@ -0,0 +1,122 @@
+import { ObjectId } from '@mikro-orm/mongodb';
+import { Injectable } from '@nestjs/common';
+import { EntityNotFoundError } from '@shared/common';
+import { Account, EntityId } from '@shared/domain';
+import { AccountRepo } from '../repo/account.repo';
+import { AccountEntityToDtoMapper } from '../mapper';
+import { AccountDto, AccountSaveDto } from './dto';
+import { AbstractAccountService } from './account.service.abstract';
+
+@Injectable()
+export class AccountServiceDb extends AbstractAccountService {
```

Comment:
We have some business logic inside this service (e.g. password hashing), which is out of scope for the repo layer. Yet, it is still quite anemic agreed. Again, since we consider database representation of accounts a temporary solution, we may keep it this way. A re-organization of the future domain objects (DTOs at the moment), however could be addressed. I'll have a look at account.entity, IAccount, and similar.